app/views/candidate_interface/course_choices/ucas.html.erb

```diff
@@ -1,13 +1,12 @@
 <% content_for :title, t('page_titles.not_eligible_yet') %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_course_choices_provider_path) %>
```

Comment:
```javascript:history.back()```
Is this better than nothing? 🤔

Reply:
👍

Reply:
Is not supposed to work without JS too?

Reply:
I am going to revert back the code, the main point of this was to update the wording.